src/components/not-found.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { NimbleEmoji } from '.'
+
+export default class NotFound extends React.PureComponent {
+  constructor(props) {
+		super(props)
+		this.state = { errored: false }
+
+		this.handleError = this.handleError.bind(this)
+	}
+
+	defaultImage() {
+		var { data, emojiProps } = this.props
+
+		return (
+			<div>
+				{NimbleEmoji({
+					data: data,
+					...emojiProps,
+					size: 38,
+					emoji: 'sleuth_or_spy',
+					onOver: null,
+					onLeave: null,
+					onClick: null,
+				})}
+			</div>
```

Comment:
Do we need this `<div>`? I can only see this ever getting called in `render()` below and I think there we don't.

We can probably just return the `NimbleEmoji({...})` component.